settings.gradle

```diff
@@ -1,5 +1,6 @@
 pluginManagement {
     plugins {
+        id 'com.android.library' version '7.3.0'
```

Comment:
Suggestion (💡): Consider updating to the latest [7.3.1](https://maven.google.com/web/index.html?q=gradle#com.android.tools.build:gradle:7.3.1) version of `com.android.library`, that is of course, unless you found an issue and couldn't do that in the first place.

PS: You might want to update `lint-baseline.xml` again after applying this change (see `by="lint 7.3.0"`).